contributing/STEERING-COMMITTEE.md

```diff
@@ -5,65 +5,156 @@ weight: 40
+was necessary for the project to ensure velocity. As the project grows and
+matures the KSC will, from time to time, consider if this policy should be
+changed.
+
+The current membership of the committee is currently (listed alphabetically by name):
+
+| &nbsp;                                                   | Member         | Organization | Profile                                  |
+| -------------------------------------------------------- | -------------- | ------------ | ---------------------------------------- |
+| <img width="30px" src="https://github.com/dewitt.png">   | DeWitt Clinton | Google       | [@dewitt](https://github.com/dewitt)     |
+| <img width="30px" src="https://github.com/mchmarny.png"> | Mark Chmarny   | Google       | [@mchmarny](https://github.com/mchmarny) |
+| <img width="30px" src="https://github.com/isdal.png">    | Tomas Isdal    | Google       | [@isdal](https://github.com/isdal)       |
+|   | TBD | Google   |        |
+|   | TBD | IBM      |        |
+|   | TBD | Pivotal  |        |
+|   | TBD | Red Hat  |        |
+
+There are currently four unfilled seats, as indicated by TBD.
+
+### Allocation of seats
+
+Seats on the steering committee are allocated based upon contribution
+to the project by an organization. No final decision has been made on the exact
+formula.
+
+As the project continues to grow, we expect to add additional seats to the
+committee, ensuring that those contributing to the project are properly
+represented.
+
+- After a seat is allocated to an organization, the organization shall nominate
+  a candidate to be confirmed by KSC. The committee reserves the right to not
+  confirm a candidate, in which the organization would need to nominate a new
+  candidate
+- Members of the committee may step down at any time. When a member steps down,
+  their organization shall nominate a new candidate.
+- If a member leaves their organization, the organization must nominate a new
+  committee member to replace them following the nomination and confirmation
+  process.
+- If an organization is unable to seat a candidate, the KSC reserves the right
+  to reallocate the seat to another organization.
+- Changes to the number of seats, which company seats are allocated to, and
+  nominations to the committee are confirmed by majority vote of the committee
+  members.
+- In situations where the organization which holds a seat is no longer a viable
+  entity (e.g. merger, dissolution, bankruptcy) the KSC will make a decision on
+  how to reallocate that seat. Seats do not automatically transfer to any
+  organization.
+- Members on the committee have a 1 year term from their confirmation, and may
+  be reconfirmed to the committee at the end of their term.
+
+## Decision process
+
+The steering committee desires to always reach consensus.
+
+Decisions are made in meetings when a quorum of the members are present and may
+pass with majority of the committee supporting it.
```

Comment:
We should be a bit more clear here... is it 50+% or 50%. And is it based on % of who votes or total members? 

I'd recommend, 50+% of members who vote - no-votes/abstains are not counted.

Reply:
As written, it's a simple-majority of the committee members, which with an odd number of members will always need to be 50+%.